progmeter.go

```diff
@@ -118,7 +118,7 @@ func (p *ProgMeter) AddEntryWithState(state, key, name, inf string) {
 }
 
 func (it *Item) Print(state string) {
-	fmt.Printf("\r[%s] %s%s", state, rightPad(it.Name, 35), it.Info)
+	fmt.Printf("\r[%s] %s%s", state, rightPad(it.Name, 40), it.Info)
```

Comment:
40 seems like a magic number. Maybe extract it to a string constant.

Probably also want to make `rightPad` into its own module.

Also thinking that we should add some tests, set up CI, and make sure they run properly on all supported platforms.

Reply:
Well, nevermind then. :(